config-xml/pom.xml

```diff
@@ -18,4 +18,35 @@
         <module>addon</module>
         <module>tests</module>
     </modules>
+
+    <build>
+        <plugins>
+            <!-- Validate the XSD. -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>xml-maven-plugin</artifactId>
+                <version>1.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>validate</goal>
+                        </goals>
+                        <phase>process-resources</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <validationSets>
+                        <validationSet>
+                            <includes>
+                                <include>dummy.xml</include>
```

Comment:
Because you only want to read the XSD, not to validate anything really. But without include, it would scan the file tree. This way I believe it does not. (Perhaps a micro-optimization not worth it)